src/pages/components/loading/usage.mdx

```diff
@@ -13,11 +13,20 @@ Loading spinners are used when retrieving data or performing slow computations,
 <AnchorLinks>
 
 <AnchorLink>Overview</AnchorLink>
-<AnchorLink>Size</AnchorLink>
+<AnchorLink>Live demo</AnchorLink>
+<AnchorLink>Formatting</AnchorLink>
 <AnchorLink>Feedback</AnchorLink>
 
 </AnchorLinks>
 
+## Overview
+
+Loading spinners are used when retrieving data or performing slow computations. They notify to the user that their request is being processed. Although they do not provide details about what is occurring on the back-end, they reassure the user that their action is being processed.
+
+It is best practice to use a loading spinner whenever the wait time is anticipated to be longer than three seconds.
```

Comment:
This could probably just be "Use a loading spinner when..." but it's fine as is.